README.rst

```diff
@@ -248,7 +248,6 @@ You can cite our work on the various libraries under the Blosc umbrella as follo
     note = {https://blosc.org}
   }
 
-
 ----
```

Comment:
Perhaps remove the horizontal rule?

I don't think it improves readability and it doesn't look nice.